packages/renderer/src/index.test.js

```diff
@@ -174,7 +174,7 @@ describe('HTMLRenderer', () => {
         const wrapper = render(<HTMLRenderer state={c} plugins={plugins} />)
         it('should pass', () => {
           expect(wrapper.html()).toEqual(
-            '<div class="ory-row"><div class="ory-cell ory-cell-sm-12 ory-cell-xs-12"><div class="ory-cell-inner ory-cell-leaf"><div><p style="text-align:center;">Asdfg</p></div></div></div></div>'
+            '<div class="ory-cell ory-cell-sm-12 ory-cell-xs-12"><div class="ory-cell-inner ory-cell-leaf"><div><p style="text-align:center">Asdfg</p></div></div></div>'
```

Comment:
What is the reason for the row being removed from the output here? A cell always needs to be wrapped in a row, so this seems strange.